Tests/FluentTests/CredentialTests.swift

```diff
@@ -0,0 +1,98 @@
+import XCTFluent
+import XCTVapor
+import Fluent
+import Vapor
+
+final class CredentialTests: XCTestCase {
+
+    func testCredentialsAuthentication() throws {
+        let app = Application(.testing)
+        defer { app.shutdown() }
+
+        // Setup test db.
+        let test = ArrayTestDatabase()
```

Comment:
Could we refactor this to be named `testDb` or something? It took me a couple of double-takes to realize this var was actually the database.

Reply:
Done